documentation/API-authentication-token.md

```diff
@@ -1,12 +1,17 @@
 # API authentication token
 
-< to be described >
+When requesting a token on the oauth2 token resource using the client_credentials a queryparameter ‘digideliveryid’ containing the identifier for the school. A scope to access data can optionally be requested and if consent has been given, will be added as a scope in the token defined below. A tokenrequest for a token used for requesting consent for a new scope does not need to contain a requested scope.
 
-jwt token with as minimum the following attributes:
+Signed JWT with as minimum the following attributes:
 
-* scopes
-* party identity
-* school identity (optional)
+client_id : identifier of application that has request this token (recipient)
+scopes : list of scopes that client_id wants to access
+
+optional (proposal):
+schoolidentifier: requested digideliveryid 
+parentidentifier: digideliveryid of parent organisation
```

Comment:
Probably needed, but would be nice to get the functional group to define what this ID is.